### PR TITLE
Fix Issue #2528

### PR DESCRIPTION
**Pull request by from-nowhere**
Fix a deprecation warning in `sklearn.linear_model.LassoLarsIC`.

The deprecation concerns the `normalize` keyword, which is currently equal to `True` by default. In order to replicate the current behaviour _after_ deprecation, we follow sklearn's [documentation on LassoLarsIC](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LassoLarsIC.html) and do the following:

1. Take care of normalization with a `sklearn.preprocessing.StandardScaler`
2. Explicitly pass `normalize=False` to the `LassoLarsIC`

From the resulting pipeline, the Lasso object can be recovered with `model[1]`.

### CODE REVIEW

**Review by TomPham97 — APPROVED** [review]

**Comment by thatlittleboy** [comment]
Duplicated by https://github.com/slundberg/shap/pull/2943
